app/src/main/java/com/quran/labs/androidquran/ui/util/TypefaceManager.kt

```diff
@@ -36,6 +37,14 @@ object TypefaceManager {
     }
   }
 
+  fun getDyslexicTypeface(context: Context): Typeface {
+    return arabicDyslexicTypeface ?: run {
```

Comment:
nit, should we rename this to dyslexicTypeface, since it's for the non-RTL fonts. also, does this font work for other languages besides English?

Reply:
Thank you for taking your time to review @ahmedre 

I have checked the font for translations with accent characters (Dutch, German, French) and those with different alphabet (Russian) and the results seem similar to the usual font. See the comparison below


<p float="center">
 					<img src="https://user-images.githubusercontent.com/98877504/174468874-cd8fedab-e3ca-4767-becb-43800e5a283d.jpg" width="100" />
  					<img src="https://user-images.githubusercontent.com/98877504/174468875-3da98d3b-ccbf-4d9b-8146-0c4b1912c22b.jpg" width="100" /> 
				   </p>